### ISSUE TITLE
Fping Notifications

### ISSUE THREAD

**Issue by sowie2016** [netdata/netdata#5605]
Got Netdata Configure and setup with Telegram Notifications.

Added fping and set up hosts all good. Blocked netdata ip and get the alarm to report critical issue packet loss but I dont recieve any notifications for this.

Am I missing a step to set up notifications for FPING?

Thanks

**Comment by ilyam8** [comment]
Hi @sowie2016 	

I see there are 4 alarms for fping
> https://github.com/netdata/netdata/blob/master/health/health.d/fping.conf

All of them are applied to every host.

what i did:
 - added '8.8.8.8' host to `fping.conf`
 - added a deny rule on my router (drop icmp dst 8.8.8.8)

`host_reachable` alarm is up

Keep in mind that default group for all those alarms is `sysadmin`, maybe you need to change it.

**Comment by sowie2016** [comment]
Hmmm........ Seems I do have all up and correct.

Set the IP'S in the /etc/hosts/ file of the servers I want to monitor

But for some reason netdata reports no packets returned. Servers currently reporting as down.

If i run from command line I can ping the same server so doesnt appear to be firewall related.

I am running on a rasperry pi any help would be appreciated.

Thanks

**Comment by ilyam8** [comment]
run plugin in debug mode

```
[netdata@homePC plugins.d]$ ./fping.plugin 
2019-03-12 00:18:34: fping.plugin: INFO: Loading config file '/opt/netdata/usr/lib/netdata/conf.d/fping.conf'...
2019-03-12 00:18:34: fping.plugin: WARNING: Cannot find file '/opt/netdata/etc/netdata/fping.conf'.
2019-03-12 00:18:34: fping.plugin: INFO: starting fping: /bin/fping -N -l -Q 1 -p 200 -R -b 56 -i 1 -r 0 -t 5000 8.8.8.8 8.8.4.4
/bin/fping: warning: timeout (-t) value larger than period (-p) produces unexpected results
CHART fping.8_8_8_8_packets '' 'FPing Packets for host 8.8.8.8' packets '8_8_8_8' fping.packets line 110020 1
DIMENSION xmt sent absolute 1 1
DIMENSION rcv received absolute 1 1
BEGIN fping.8_8_8_8_packets
SET xmt = 5
SET rcv = 5
END
CHART fping.8_8_8_8_quality '' 'FPing Quality for host 8.8.8.8' percentage '8_8_8_8' fping.quality area 110010 1
DIMENSION returned '' absolute 1 1
BEGIN fping.8_8_8_8_quality
SET returned = 100
END
CHART fping.8_8_8_8_latency '' 'FPing Latency for host 8.8.8.8' ms '8_8_8_8' fping.latency area 110000 1
DIMENSION min minimum absolute 10 1000
DIMENSION max maximum absolute 10 1000
DIMENSION avg average absolute 10 1000
BEGIN fping.8_8_8_8_latency
SET min = 810
SET avg = 834
SET max = 863
END
CHART fping.8_8_4_4_packets '' 'FPing Packets for host 8.8.4.4' packets '8_8_4_4' fping.packets line 110020 1
DIMENSION xmt sent absolute 1 1
DIMENSION rcv received absolute 1 1
BEGIN fping.8_8_4_4_packets
SET xmt = 5
SET rcv = 5
END
CHART fping.8_8_4_4_quality '' 'FPing Quality for host 8.8.4.4' percentage '8_8_4_4' fping.quality area 110010 1
DIMENSION returned '' absolute 1 1
BEGIN fping.8_8_4_4_quality
SET returned = 100
END
CHART fping.8_8_4_4_latency '' 'FPing Latency for host 8.8.4.4' ms '8_8_4_4' fping.latency area 110000 1
DIMENSION min minimum absolute 10 1000
DIMENSION max maximum absolute 10 1000
DIMENSION avg average absolute 10 1000
BEGIN fping.8_8_4_4_latency
SET min = 829
SET avg = 839
SET max = 845
END
^C8.8.8.8 : xmt/rcv/%loss = 8/8/0%, min/avg/max = 8.10/8.39/8.63
8.8.4.4 : xmt/rcv/%loss = 8/8/0%, min/avg/max = 8.18/8.42/8.65
```

___

> If i run from command line I can ping the same server so doesnt appear to be firewall related.

check it with tcpdump

**Comment by sowie2016** [comment]
As basic as it is I think my host file must be wrong....

Can you give me an example of yours for extenal IP's so I can compare please?

**Comment by sowie2016** [comment]
Scrap that further looking I'd listed my hosts as host 1 with a space.

Thank you for your time

**Comment by ilyam8** [comment]
> Can you give me an example of yours for extenal IP's so I can compare please?

By external IPs you mean fping hosts, don't you?

this is mine

> hosts="8.8.8.8 8.8.4.4"

i just added them to test :smile: 	


> Scrap that further looking I'd listed my hosts as host 1 with a space.

Have you resolved the issue?


If no, please run `fping.plugin` in [debug mode](https://github.com/netdata/netdata/issues/5605#issuecomment-471735154) and post here your result

**Comment by sowie2016** [comment]
Yes working Perfect.

Thank You
